.travis.yml

```diff
@@ -64,8 +64,10 @@ before_script:
 script:
   - export CONTINUE=1
   - if [ $SECONDS -gt 1200 ]; then export CONTINUE=0; fi  # Likely the depends build took very long
+  - if [ $TRAVIS_REPO_SLUG = "bitcoin/bitcoin" ]; then export CONTINUE=1; fi  # Whitelisted repo (90 minutes build time)
```

Comment:
The normal job timeout is 50 minutes, and we check if more than 20 minutes (1200 seconds) have passed. So with this extended timeout, should we check `$SECONDS -gt 3600`? Is there a way to directly check the timeout value, rather than relying on a hard coded repo name?

Reply:
I couldn't find the timeout in https://docs.travis-ci.com/user/environment-variables/#default-environment-variables